tsfile/src/main/java/org/apache/iotdb/tsfile/read/TsFileSequenceReader.java

```diff
@@ -665,6 +665,100 @@ protected int binarySearchInTimeseriesMetadataList(
     return paths;
   }
 
+  /**
+   * @return an iterator of timeseries list, in which names of timeseries are ordered in dictionary order
+   * @throws IOException io error
+   */
+  public Iterator<List<Path>> getPathsIterator() throws IOException {
```

Comment:
Where is this method called, or this one will be used in later PRs?

Reply:
This method will be called by compaction module. Thanks for your kindly remind.